https://github.com/bazelbuild/rules_rust/pull/509

diff --git a/rust/platform/triple_mappings.bzl b/rust/platform/triple_mappings.bzl
index f686416..6533598 100644
--- a/rust/platform/triple_mappings.bzl
+++ b/rust/platform/triple_mappings.bzl
@@ -9,11 +9,11 @@ _CPU_ARCH_TO_BUILTIN_PLAT_SUFFIX = {
     "i686": "x86_32",
     "s390x": "s390x",
     "asmjs": None,
-    "i386": None,
+    "i386": "i386",
     "i586": None,
     "powerpc64": None,
     "powerpc64le": None,
-    "armv7": None,
+    "armv7": "armv7",
     "armv7s": None,
     "s390": None,
     "le32": None,
@@ -37,7 +37,7 @@ _SYSTEM_TO_BUILTIN_SYS_SUFFIX = {
     "bitrig": None,
     "dragonfly": None,
     "netbsd": None,
-    "openbsd": None,
+    "openbsd": "openbsd",
     "solaris": None,
 }
 